configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionStateWatcher.java

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.config.server.session;
+
+import java.util.Optional;
+import java.util.logging.Level;
+import com.yahoo.text.Utf8;
+import com.yahoo.vespa.config.server.ReloadHandler;
+import com.yahoo.vespa.config.server.monitoring.MetricUpdater;
+import com.yahoo.vespa.curator.Curator;
+import org.apache.curator.framework.recipes.cache.ChildData;
+
+import java.util.concurrent.Executor;
+import java.util.logging.Logger;
+
+/**
+ * Watches one particular session (/config/v2/tenants/&lt;tenantName&gt;/sessions/&lt;n&gt;/sessionState in ZooKeeper)
+ * The session must be in the session repo.
+ *
+ * @author Vegard Havdal
+ */
```

Comment:
I suggest to add yourself as author since you have created the class/file.

Reply:
Yes, good point, will do